sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/configuration/login/Action.java

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sharedkernel.configuration.login;
+
+import java.util.Optional;
+
+import com.daimler.sechub.adapter.ActionType;
+import com.daimler.sechub.adapter.SecHubTimeUnit;
+
+public class Action {
+
+    ActionType type;
+
+	Optional<String> selector = Optional.empty();
+
+	Optional<String> value = Optional.empty();
+	
+	Optional<String> description = Optional.empty();
+	
+	Optional<SecHubTimeUnit> unit = Optional.empty();
+
+    /**
+	 * @return type of action as string (e.g. "input", "click")
```

Comment:
shouldn't it be more like:

```
public String getType() {
		return type.toString();
}
```

?

Reply:
Thanks for pointing this out. I removed the comment. The getter should return an `ActionType`.